.spelling

```diff
@@ -317,6 +318,7 @@ istio-system
 istio.io
 istio.io.
 Istiofied
+istiod
```

Comment:
Shouldn't need this. The usual convention is to use uppercase (Istiod) when referring to components generically, and use back-ticks around the lowercase name (`istiod`) when referring specifically to the deployment. I see that this document is using backticks in most places, so it would probably be better to change them to upercase Istiod, but not critical IMO.
```suggestion
```

Reply:
Ok, I'll change my `istiod`s to Istiods throughout.